project-base/storefront/components/Blocks/SimpleNavigation/SimpleNavigationListItem.tsx

```diff
@@ -4,12 +4,12 @@ import { getStringWithoutTrailingSlash } from 'helpers/parsing/stringWIthoutSlas
 import { twMergeCustom } from 'helpers/twMerge';
 import { PageType } from 'store/slices/createPageLoadingStateSlice';
 import { twJoin } from 'tailwind-merge';
-import { ListedItemPropType } from 'types/simpleNavigation';
+import { ListedItemPropType, ListedItemPropTypeTypename } from 'types/simpleNavigation';
 
 type SimpleNavigationListItemProps = {
     listedItem: ListedItemPropType;
     imageType?: string;
-    linkType: PageType;
+    linkType?: PageType;
```

Comment:
Since this essentially becomes an override, I would choose a name that describes that. Could you rename it to something which tells the developer that he can enforce/override the type, which is otherwise calculated inside the component?